relay-event-normalization/src/normalize/mod.rs

```diff
@@ -116,6 +118,7 @@ impl<'a> StoreNormalizeProcessor<'a> {
     fn normalize_spans(&self, event: &mut Event) {
         if event.ty.value() == Some(&EventType::Transaction) {
             normalize_app_start_spans(event);
+            normalize_all_metrics_summaries(event);
```

Comment:
This function also modifies `event._metrics_summary`, so we should move it out of `normalize_spans` IMO.